reporters_db/data/reporters.json

```diff
@@ -9604,6 +9604,27 @@
             "variations": {}
         }
     ],
+    "Kelly": [
+        {
+            "cite_type": "state",
+            "editions": {
+                "Kelly": {
+                    "end": "1849-12-31T00:00:00",
+                    "start": "1847-01-01T00:00:00"
+                }
+            },
+            "examples": [
+                "1 Kelly 355"
+            ],
+            "mlz_jurisdiction": [
+                "us:ga;supreme.court"
+            ],
+            "name": "Kelly's Reports",
+            "variations": {
+                "Ga. (Kelly)": "Kelly"
```

Comment:
I might be wrong here, but this variation feels like it should be on Georgia Decisions not on the nominative reporter.

Reply:
Move just the `"variations:"` section but keep the main `"Kelly"` reporter? If so, that makes more sense to me. I can make that change if you confirm.

Reply:
I went looking for an example to back up my idea, and I failed to find any.  But to clarify I would expect, Ga. (Kelly) to be a variation of the "Ga." Reporter not the Kelly one.  

If it was Kelly (Ga.) I would feel the opposite.  

I would make `"variations": {}` on Kelly but certainly keep the reporter.  @mlissner do you have thoughts.

Reply:
Okay. I pushed a commit to the branch that does that.

Reply:
Thanks @lmullen